DotNetHowToAutocomplete/DotNetHowToAutocomplete/Controllers/HomeController.cs

```diff
@@ -38,40 +38,27 @@ public ActionResult IndexJavaScript()
             return View();
         }
 
-        public ActionResult About()
-        {
-            ViewBag.Message = "Your application description page.";
-
-            return View();
-        }
-
-        public ActionResult Contact()
-        {
-            ViewBag.Message = "Your contact page.";
-
-            return View();
-        }
         public ActionResult Suggest(bool highlights, bool fuzzy, string term)
         {
             InitSearch();
 
             // Call suggest API and return results
-            SuggestParameters sp = new SuggestParameters()
+            SuggestParameters ap = new SuggestParameters()
```

Comment:
It should be `sp` for `SuggestParameters` and `ap` for `AutocompleteParameters`

Reply:
Just pushed an update.  Should have covered these comments and found some you missed!  Thanks!